.changeset/green-days-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+"infrastructure": patch
```

Comment:
**question**: why do you think this is a `patch`?

Reply:
just changed from patch to minor with Mauro in call